AccountService/src/LoginService.java

```diff
@@ -1,21 +1,21 @@
 import java.util.ArrayList;
 
 public class LoginService {
-    CustomerService customerService;
+	CustomerService customerService;
 
-    public LoginService(CustomerService customerService) {
-        this.customerService = customerService;
-    }
+	public LoginService(CustomerService customerService) {
```

Comment:
As a good practice, all method parameters should be set to `final` https://stackoverflow.com/questions/500508/why-should-i-use-the-keyword-final-on-a-method-parameter-in-java